src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/AbstractPtvInTest.java

```diff
@@ -0,0 +1,220 @@
+import fi.metatavu.kuntaapi.server.rest.model.WebPage;
+import fi.metatavu.ptv.client.model.V2VmOpenApiDailyOpeningTime;
+import fi.metatavu.ptv.client.model.V4VmOpenApiPhone;
+import fi.metatavu.ptv.client.model.V4VmOpenApiPhoneSimple;
+import fi.metatavu.ptv.client.model.V4VmOpenApiServiceHour;
+import fi.metatavu.ptv.client.model.V7VmOpenApiAddressWithMovingIn;
+import fi.metatavu.ptv.client.model.VmOpenApiAddressPostOfficeBoxIn;
+import fi.metatavu.ptv.client.model.VmOpenApiAddressStreetWithCoordinatesIn;
+import fi.metatavu.ptv.client.model.VmOpenApiAreaIn;
+import fi.metatavu.ptv.client.model.VmOpenApiLanguageItem;
+import fi.metatavu.ptv.client.model.VmOpenApiLocalizedListItem;
+import fi.metatavu.ptv.client.model.VmOpenApiWebPageWithOrderNumber;
+import fi.otavanopisto.kuntaapi.server.integrations.ptv.translation.PtvAddressSubtype;
+import fi.otavanopisto.kuntaapi.test.AbstractIntegrationTest;
+
+public class AbstractPtvInTest extends AbstractIntegrationTest {
+
+  protected List<WebPage> createWebPages(String language, String type, String url, String value, String description) {
+    WebPage result = new WebPage();
+    result.setDescription(description);
+    result.setLanguage(language);
+    result.setType(type);
+    result.setUrl(url);
+    result.setValue(value);
+    
+    return Arrays.asList(result);
+  }
+
+  protected List<V4VmOpenApiPhone> createPtvInPhones(String language, String type, String prefixNumber, String number, String serviceChargeType, String chargeDescription, Boolean isFinnishServiceNumber, String additionalInformation) {
+    V4VmOpenApiPhone result = new V4VmOpenApiPhone();
+    result.setAdditionalInformation(additionalInformation);
+    result.setChargeDescription(chargeDescription);
+    result.setIsFinnishServiceNumber(isFinnishServiceNumber);
+    result.setLanguage(language);
+    result.setNumber(number);
+    result.setPrefixNumber(prefixNumber);
+    result.setServiceChargeType(serviceChargeType);
+    return Arrays.asList(result);
+  }
+
+  protected List<V4VmOpenApiPhoneSimple> createPtvInFaxNumbers(String language, String prefixNumber, String number, Boolean isFinnishServiceNumber) {
+    V4VmOpenApiPhoneSimple result = new V4VmOpenApiPhoneSimple();
+    result.setIsFinnishServiceNumber(isFinnishServiceNumber);
+    result.setLanguage(language);
+    result.setNumber(number);
+    result.setPrefixNumber(prefixNumber);
+    return Arrays.asList(result);
+  }
+
+  protected List<Phone> createPhones(String language, String type, String prefixNumber, String number, String serviceChargeType, String chargeDescription, Boolean isFinnishServiceNumber, String additionalInformation) {
+    Phone result = createPhone(language, type, prefixNumber, number, serviceChargeType, chargeDescription,
+        isFinnishServiceNumber, additionalInformation);
+
+    return Arrays.asList(result);
+  }
+  
+  protected Phone createPhone(String language, String type, String prefixNumber, String number,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)